src/integrationTest/java/uk/gov/hmcts/darts/utils/RedisConfiguration.java

```diff
@@ -7,7 +7,7 @@
 
 @TestConfiguration
 class RedisConfiguration {
-    public static final RedisServer REDISSERVER = new RedisServer();
+    public static final RedisServer REDISSERVER = new RedisServer(6380);
```

Comment:
I wanted to disassociated the integration test redis config from the local docker based deployment. This change allows that to happen, otherwise you have to shut down you're docker instance before being able to run the tests.